docs/.vuepress/public/presets/toml/tokyo-night.toml

```diff
@@ -1,64 +1,205 @@
 format = """
-[░▒▓](#a3aed2)\
-[  ](bg:#a3aed2 fg:#090c0c)\
-[](bg:#769ff0 fg:#a3aed2)\
+[ ](bg:blurple_darkest)\
```

Comment:
I would prefer staying with `░▒▓` to avoid changing the preset too much.

Reply:
I reverted it like you requested. I do think it looks "worse" though, especially in some terminals with specific fonts, and that the colored rectangles do a better job of matching the original author's screenshot across different environments.